sotopia/database/serialization.py

```diff
@@ -14,66 +15,66 @@
 
 
 class TwoAgentEpisodeWithScenarioBackgroundGoals(BaseModel):
-    episode_id: str = Field(required=True)
-    environment_id: str = Field(required=True)
-    agent_ids: list[str] = Field(required=True)
-    experiment_tag: str = Field(required=True)
-    experiment_model_name_pairs: list[str] = Field(required=True)
-    raw_messages: list[list[tuple[str, str, str]]] = Field(required=True)
-    raw_rewards: list[tuple[float, dict[str, float]] | float] = Field(required=True)
-    raw_rewards_prompt: str = Field(required=True)
-    scenario: str = Field(required=True)
-    codename: str = Field(required=True)
-    agents_background: dict[str, str] = Field(required=True)
-    social_goals: dict[str, str] = Field(required=True)
-    social_interactions: str = Field(required=True)
-    reasoning: str = Field(required=False)
-    rewards: list[tuple[float, dict[str, float]]] = Field(required=False)
+    episode_id: str = Field()
```

Comment:
what's the default requirement now? is it still required=True.

Reply:
Check out here: https://docs.pydantic.dev/latest/migration/#required-optional-and-nullable-fields. I think I maintained all of the semantics